packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -11,7 +11,9 @@
 
             <p data-translate="{{:: 'pci_project_flavors_category_definition_' + flavorByCategory.category }}"></p>
 
-            <div class="container-fluid px-0">
+            <p data-ng-if="flavorByCategory.flavors.length === 0" data-translate="pci_project_flavors_category_definition_none"></p>
+
+            <div class="container-fluid px-0" data-ng-if="flavorByCategory.flavors.length > 0">
```

Comment:
Not yours but removing default padding from the container doesn't make sens `container-fluid px-0`.

Reply:
why ?